data_loader/8_fakenews/fakenews_plugin.py

```diff
@@ -0,0 +1,167 @@
+"""Load data from the Squirro Fake News Training API"""
+
+import logging
+import hashlib
+import json
+import requests
+import urllib.parse
+from squirro.dataloader.data_source import DataSource
```

Comment:
While you have an isort config file in the folder, these imports don't seem to be isorted.

Reply:
no clue, black is active... have sorted manually for now.

Reply:
yup, learned something today